cloud_service/installing_cloud_ocp/install-secured-cluster-cloud-ocp.adoc

```diff
@@ -9,12 +9,9 @@ toc::[]
 [role="_abstract"]
 You can install {product-title-managed-short} on your secured clusters by using the Operator.
```

Comment:
Should we move this under "Installing RHACS on secured clusters by using the Operator"? OR perhaps even remove this sentence because it looks like a repetition of the title? WDYT?

Reply:
As I understand it, the abstract role is required by the [doc template](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc). "[role="_abstract"]
This paragraph is the assembly introduction. It explains what the user will accomplish by working through the modules in the assembly and sets the context for the user story the assembly is based on."

Reply:
I removed the section underneath since it was repeated info, though - good suggestion.